src/ci/docker/cross/install-mips-musl.sh

```diff
@@ -10,15 +10,10 @@
 
 set -ex
 
-mkdir /usr/local/mips-linux-musl
-
-# originally from
-# https://downloads.openwrt.org/snapshots/trunk/ar71xx/generic/
-# OpenWrt-Toolchain-ar71xx-generic_gcc-5.3.0_musl-1.1.16.Linux-x86_64.tar.bz2
+# This file was generated with https://github.com/richfelker/musl-cross-make and
+# http://www.nongnu.org/libunwind/. The cross compiler in the sysroot was
```

Comment:
Oh. So do we actually know unwinding works in Rust static executables on mips with the nongnu library?